Kajabi/sage-lib
# [DSS-248] Icons: Update icon font (9 new)

## Description
Adds the following new icons to Sage:

|  Name  |   |
|--------|--------|
|`certificate`|![Screen Shot 2022-12-14 at 4 56 14 PM](https://user-images.githubusercontent.com/1175111/207747425-70ad579f-5541-43c0-b425-ba369b3d82d3.png)|
|`creator-studio`|![Screen Shot 2022-12-14 at 4 56 30 PM](https://user-images.githubusercontent.com/1175111/207747443-bd15d405-b7a4-4568-8e4e-bdbdd55b35a6.png)|
|`creator-studio-filled`|![Screen Shot 2022-12-14 at 4 56 40 PM](https://user-images.githubusercontent.com/1175111/207747473-aca6ed01-7ee1-426f-920d-a952a5940d99.png)|
|`editor`|![Screen Shot 2022-12-14 at 4 56 52 PM](https://user-images.githubusercontent.com/1175111/207747486-ba480e6f-c4f5-4073-80f6-4106337e522c.png)|
|`editor-filled`|![Screen Shot 2022-12-14 at 4 57 02 PM](https://user-images.githubusercontent.com/1175111/207747504-02fae0ad-212e-4b92-a5bf-a5648adcefd3.png)|
|`emoji`|![Screen Shot 2022-12-14 at 4 57 09 PM](https://user-images.githubusercontent.com/1175111/207747527-3657ef97-57dd-4a7f-9311-fcc9d129192a.png)|
|`marker`|![Screen Shot 2022-12-14 at 4 57 25 PM](https://user-images.githubusercontent.com/1175111/207747587-6ba2e263-ad4e-46c8-8cba-f135afe7aca6.png)|
|`marker-filled`|![Screen Shot 2022-12-14 at 4 57 33 PM](https://user-images.githubusercontent.com/1175111/207747622-4382c3d7-b471-447a-9470-b4ca3da75f40.png)|
|`text-styles`|![Screen Shot 2022-12-14 at 4 57 49 PM](https://user-images.githubusercontent.com/1175111/207747641-7f173de4-8e94-4254-a60f-db88ea08d88b.png)|

## Testing in `sage-lib`
1. Navigate to http://localhost:4000/pages/foundations/icon
2. Verify new icons are added to the system
3. Verify all icons are rendering as expected


## Testing in `kajabi-products`
1. (**LOW**) Updated icon font w/ 9 new icons. No impact is expected.


## Related
https://kajabi.atlassian.net/browse/DSS-248
